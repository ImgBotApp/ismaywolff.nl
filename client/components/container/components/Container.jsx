import styled from 'styled-components'

const Container = styled.div`
  margin: 0 auto;
  padding: 1rem;
  max-width: 30rem;
`

export default Container
