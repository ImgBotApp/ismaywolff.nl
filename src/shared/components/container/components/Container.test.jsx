/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import Container from './Container'

describe('<Container />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Container />)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
