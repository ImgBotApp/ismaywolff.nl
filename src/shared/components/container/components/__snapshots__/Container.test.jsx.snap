// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> renders correctly 1`] = `
.bZScRv {
  margin: 0 auto;
  max-width: 30rem;
  padding: 1rem;
}

<div
  className="sc-bdVaJa bZScRv"
/>
`;
